user-context.md

```diff
@@ -20,6 +20,10 @@ web:
     uri: "/me"
 ```
 
+### Authentication
+
+This endpoint requires authentication, and accepts all forms of authentication. 
```

Comment:
API credential Basic Auth is always associated with a user, so you'd be authenticated against that user. 

So Basic Auth as stated in https://github.com/stormpath/stormpath-framework-spec/pull/75/files#diff-940526024ff9cd3b0d3e9eae41005349R48